rmgpy/rmg/model.py

```diff
@@ -673,29 +681,32 @@ def addNewSurfaceObjects(self,obj,newSurfaceSpecies,newSurfaceReactions,reaction
         moves them to appropriate newSurfaceSpc/RxnsAdd/loss sets
         returns false if the surface has changed
         """
-        surfSpcs = set(self.surface.species)
-        surfRxns = set(self.surface.reactions)
-        
+        surf_spcs = set(self.surface.species)
+        surf_rxns = set(self.surface.reactions)
+
         newSurfaceSpecies = set(newSurfaceSpecies)
         newSurfaceReactions = set(newSurfaceReactions)
-                    
-        addedRxns = {k for k in obj if isinstance(k,Reaction)}
-        addedSurfaceRxns = newSurfaceReactions - surfRxns
-                    
-        addedBulkRxns = addedRxns-addedSurfaceRxns
-        lostSurfaceRxns = (surfRxns - newSurfaceReactions) | addedBulkRxns
-                    
-        addedSpcs = {k for k in obj if isinstance(k,Species)} | {k.getMaximumLeakSpecies(reactionSystem.T.value_si, reactionSystem.P.value_si) for k in obj if isinstance(k,PDepNetwork)}
-        lostSurfaceSpcs = (surfSpcs-newSurfaceSpecies) | addedSpcs
-        addedSurfaceSpcs = newSurfaceSpecies - surfSpcs
-                    
-        self.newSurfaceSpcsAdd = self.newSurfaceSpcsAdd | addedSurfaceSpcs
-        self.newSurfaceRxnsAdd = self.newSurfaceRxnsAdd | addedSurfaceRxns
-        self.newSurfaceSpcsLoss = self.newSurfaceSpcsLoss | lostSurfaceSpcs
-        self.newSurfaceRxnsLoss = self.newSurfaceRxnsLoss | lostSurfaceRxns
-        
-        return not (self.newSurfaceRxnsAdd != set() or self.newSurfaceRxnsLoss != set() or self.newSurfaceSpcsLoss != set() or self.newSurfaceSpcsAdd != set())
-    
+
+        added_rxns = {k for k in obj if isinstance(k, Reaction)}
+        added_surface_rxns = newSurfaceReactions - surf_rxns
+
+        added_bulk_rxns = added_rxns - added_surface_rxns
+        lost_surface_rxns = (surf_rxns - newSurfaceReactions) | added_bulk_rxns
+
+        added_spcs = {k for k in obj if isinstance(k, Species)} | {
+            k.getMaximumLeakSpecies(reactionSystem.T.value_si, reactionSystem.P.value_si) for k in obj if
+            isinstance(k, PDepNetwork)}
+        lost_surface_spcs = (surf_spcs - newSurfaceSpecies) | added_spcs
+        added_surface_spcs = newSurfaceSpecies - surf_spcs
+
+        self.newSurfaceSpcsAdd = self.newSurfaceSpcsAdd | added_surface_spcs
+        self.newSurfaceRxnsAdd = self.newSurfaceRxnsAdd | added_surface_rxns
+        self.newSurfaceSpcsLoss = self.newSurfaceSpcsLoss | lost_surface_spcs
+        self.newSurfaceRxnsLoss = self.newSurfaceRxnsLoss | lost_surface_rxns
+
+        return not (
+                    self.newSurfaceRxnsAdd != set() or self.newSurfaceRxnsLoss != set() or self.newSurfaceSpcsLoss != set() or self.newSurfaceSpcsAdd != set())
```

Comment:
I think this should either be one line or split after each `or`.

Reply:
Done, I agree this is much better